example/src/screens/APIClientScreen.tsx

```diff
@@ -54,7 +54,7 @@ const styles = StyleSheet.create({
         ...Platform.select({
             ios: { borderColor: PlatformColor("link") },
             android: {
-                borderColor: PlatformColor("?attr/colorControlNormal"),
+                // borderColor: PlatformColor("?attr/colorControlNormal"),
```

Comment:
Why the comment?

Reply:
`borderColor` is still broken on Android :(